### PR TITLE
Disable all tests for M.W32.Registry on uap

### PR DESCRIPTION
**Pull request by alexperovich**
These tests all need to be disabled because the implementation does nothing but throw PlatformNotSupported on uap

Fixes https://github.com/dotnet/corefx/issues/18652

### CODE REVIEW

**Comment by ViktorHofer** [comment]
Why were they running at all? I can't see any uap configuration

**Comment by alexperovich** [comment]
They were running because there is a netstandard configuration.

File: src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj [hunk 1]
```diff
@@ -5,9 +5,13 @@
...
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Windows_NT-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Windows_NT-Release|AnyCPU'" />
-  <ItemGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Windows_NT-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Windows_NT-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
```
Comment (danmosemsft) [inline src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj]: why are we keeping a ns configuration if it's going ot do nothing?

Comment (ViktorHofer) [inline src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj]: I talked with Alex about it. I suggested to  remove it from here and from the Configuration.props. He said he's gonna check

**Pull request merged**